src/ontology/mondo-edit.obo

```diff
@@ -372502,13 +372503,14 @@ subset: rare
 synonym: "body of uterus undifferentiated carcinoma" EXACT [MONDO:patterns/location]
 synonym: "endometrial undifferentiated carcinoma" RELATED [Orphanet:213721]
 xref: GARD:20480 {source="MONDO:GARD"}
-xref: ICD10CM:C50 {source="Orphanet:213721/ntbt", source="MONDO:relatedTo", source="Orphanet:213721"}
+xref: ICD10CM:C54.9 {source="Orphanet:213721/ntbt", source="MONDO:relatedTo", source="https://orcid.org/0000-0002-7347-2434", source="Orphanet:213721"}
```

Comment:
@katiermullen did you mean to have this x-ref as equivalent? 
(I can update the PR)

Reply:
no, it was meant to be related.